src/Nox.Reference.Data.World/Seeds/CultureDataSeeder.cs

```diff
@@ -144,23 +145,23 @@ public void Seed()
 
                 var numberFormat = new NumberFormatInfo()
                 {
-                    CurrencySymbol = GetNodeText(currencySymbolNode),
-                    DecimalSeparator = GetNodeText(decimalSeparatorNode),
-                    Digit = GetNodeText(digitNode),
-                    ExponentSeparator = GetNodeText(exponentSeparatorNode),
-                    GroupingSeparator = GetNodeText(groupingSeparatorNode),
-                    Infinity = GetNodeText(infinityNode),
-                    InternationalCurrencySymbol = GetNodeText(internationalCurrencySymbolNode),
-                    MinusSign = GetNodeText(minusSignNode),
-                    MonetaryDecimalSeparator = GetNodeText(monetaryDecimalSeparatorNode),
-                    NotANumberSymbol = GetNodeText(notANumberNode),
-                    PadEscape = GetNodeText(PadEscapeNode),
-                    PatternSeparator = GetNodeText(PatternSeparatorNode),
-                    Percent = GetNodeText(PercentNode),
-                    PerMill = GetNodeText(PerMillNode),
-                    PlusSign = GetNodeText(PlusSignNode),
-                    SignificantDigit = GetNodeText(SignificantDigitNode),
-                    ZeroDigit = GetNodeText(ZeroDigitNode),
+                    CurrencySymbol = DataSeederUtils.GetNodeText(currencySymbolNode),
```

Comment:
use static using for DataSeederUtils to cut down code.

Reply:
I think it's fine either way, but I like the explicit version more. Do you have strong feelings about that?

Reply:
I guess short version is more readable